added random default block.timestamp to .dapprc

manuelpolzhofer opened the pull request:
hey,

really nice dapp.tools template. 💯 

I think it would be good to add a random default `block.timestamp` for testing.

Otherwise, it is zero by default which might introduce potential bugs during development. (edge case which will never occur on mainnet)

transmissions11 commented:
`DAPP_TIMESTAMP` is not an environment variable DappTools accepts.

> Otherwise, it is zero by default which might introduce potential bugs during development. (edge case which will never occur on mainnet)

This is false.
<img width="397" alt="Screen Shot 2021-09-06 at 3 32 35 PM" src="https://user-images.githubusercontent.com/26209401/132263695-07f10a1a-412b-44a7-a50e-a8d23e6243e6.png">

manuelpolzhofer commented:
@transmissions11 
Ah, sorry.  You are right. Fixed it.

I also added `HEVM_ADDRESS` for the init in the test. 

I would recommend using a realistic block.timestamp for tests.  Obviously you could just do it in the `setUp` method like Maker. 

https://github.com/makerdao/dss/commit/f332ad467b431e4ee3110326fa12cc460b641d96

However, if you are new to the Solidity you might be not aware about that fact.

gakonst commented:
Hey @manuelpolzhofer thanks for the PR! 

I don't think we should merge the timestamp setting in the dapprc as it's something that I wouldn't expect as a user. 

Personally, I'd prefer having it as 0 and opting in, rather than having a random one as the default.

I'll merge the HEVM constant change if you want to remove the timestamp commit or if you want to close this and do it in a new PR!

manuelpolzhofer commented:
Okay. I opened a second one.  Maybe explicitly defining it in the tests is cleaner.
